release.sh

```diff
@@ -153,7 +165,7 @@ git pull
 
 MVN_VERSION_DEVELOP=$(xmllint --xpath "//*[local-name()='project']/*[local-name()='version']/text()" pom.xml)
 
-git checkout master
+git checkout ${BRANCH}
 
```

Comment:
Isn’t this the branch we are on already?

Reply:
Yes, good point